lib/widgets/custom_outlined_button.dart

```diff
@@ -12,22 +12,25 @@ class ActionButton extends StatelessWidget {
   final Icon icon;
   final VoidCallback? onPressed;
 
+  static const _buttonTextStyle = TextStyle(
+    fontSize: 16,
+    fontWeight: FontWeight.bold,
+    fontFamily: 'TATFont',
+  );
```

Comment:
Suggest creating mixin, since both `custom_outline_button` and `custom_elevated_button` used the same code.

If this is a considerable solution, the name of mixin might become our next nightmare.
Hence, I temporarily call it `ParrotButtonStyleMixin` now.

```dart
mixin ParrotButtonStyleMixin on Widget {
  TextStyle get buttonTextStyle => const TextStyle(
    fontSize: 16,
    fontWeight: FontWeight.bold,
    fontFamily: 'TATFont',
  );
}
```

Then update widget like this:

```diff
import 'package:flutter/material.dart';
+ import 'package:parrot_number/widgets/parrot_button_mixin.dart';

- class CustomOutlinedButton extends StatelessWidget {
+ class CustomOutlinedButton extends StatelessWidget with ParrotButtonStyleMixin {
  const CustomOutlinedButton({
    super.key,
    required this.text,
    required this.icon,
    this.onPressed,
  });

  final String text;
  final Icon icon;
  final VoidCallback? onPressed;

- static const _buttonTextStyle = TextStyle(
-   fontSize: 16,
-   fontWeight: FontWeight.bold,
-   fontFamily: 'TATFont',
- );
...
          foregroundColor: Colors.black,
-         textStyle: _buttonTextStyle,
+         textStyle: buttonTextStyle,
          side: const BorderSide(color: Colors.black, width: 2),
```

Reply:
This is a great approach for using `mixin`s!
Addressed at 8e8fec8.